src/types/motions.ts

```diff
@@ -105,10 +105,48 @@ export interface StakerReward {
   isClaimed: boolean;
 }
 
+enum DomainColor {
+  LIGHT_PINK,
+  PINK,
+  BLACK,
+  EMERALD_GREEN,
+  BLUE,
+  YELLOW,
+  RED,
+  GREEN,
+  PERIWINKLE,
+  GOLD,
+  AQUA,
+  BLUE_GREY,
+  PURPLE,
+  ORANGE,
+  MAGENTA,
+  PURPLE_GREY,
+}
```

Comment:
Can't we get this directly from the generated graphql types ?

Reply:
I don't think so, at least not right now it seems. That would be nice tho, as I see that there are other interfaces in this file that are copies of what we have in the generated graphql type file